intelmq/etc/defaults.conf

```diff
@@ -6,6 +6,7 @@
     "destination_pipeline_password": null,
     "destination_pipeline_port": 6379,
     "error_dump_message": true,
+    "error_forward_message": false,
```

Comment:
@wagner-certat i dont know if I added this by accident...

Reply:
Thanks for catching these!